test/components-pages/fixtures-deprecated/getRenderedWidgets/tests.js

```diff
@@ -5,7 +5,7 @@ var markoWidgets = require('marko-widgets');
 describe(path.basename(__dirname), function() {
     it('should serialize widget config down to the browser', function() {
 
-        expect(window.simpleWidgets).to.equal(undefined);
+        expect(window.simpleWidgets).to.eql([]);
```

Comment:
Why did this change to an empty array?

Reply:
The tests used to run before the static component code:
https://github.com/marko-js/marko/blob/ecc7763f26d8cb25f13319b7f4bcea7c6171645a/test/components-pages/fixtures-deprecated/getRenderedWidgets/components/app-simple/index.js#L2

They now run after, but it doesn't change the meaning of the tests.  It's still saying the same thing: "expect there to be no `simpleWidgets` yet."